docs/docs/scanner/vulnerability.md

```diff
@@ -307,6 +307,25 @@ By default, all relationships are included in the scan.
 !!! warning
     As it may not provide a complete package list, `--pkg-relationships` cannot be used with `--dependency-tree`, `--vex` or SBOM generation.
 
+### Detection Priority
+
+Trivy provides a `--detection-priority` flag to control the balance between false positives and false negatives in vulnerability detection.
+This concept is similar to the relationship between [precision and recall][precision-recall] in machine learning evaluation.
+
+```bash
+$ trivy image --detection-priority {precise|comprehensive} alpine:3.15
+```
+
+- `precise`: This mode prioritizes reducing false positives. It results in less noisy vulnerability reports but may miss some potential vulnerabilities.
+- `comprehensive`: This mode aims to detect more vulnerabilities, potentially including some that might be false positives.
+  It provides broader coverage but may increase the noise in the results.
+
+The default value is `precise`.
+
+Regardless of the chosen mode, user review of detected vulnerabilities is crucial:
+
+- `precise`: Review thoroughly, considering potential missed vulnerabilities.
+- `comprehensive`: Carefully investigate each reported vulnerability due to increased false positive possibility.
```

Comment:
I think we need to add `Notes` from PR description to docs.
We can also add a table with differences.
something like this:
|   | precise   | comprehensive  | 
|---|---|---|
| pkg managers  | only OS pkgs | OS pkgs + language pkgs |
| dart  | `0.0.0`  | use min version from SDK |
| gomod  | skip `stdlib`  | use min version from `toolchain` |

Reply:
Updated https://github.com/aquasecurity/trivy/pull/7288/commits/67b80d32b9adf21c8474fa8622ada35ace98ab9c